### ISSUE TITLE
0.77.0 broke my `def-env` command

### ISSUE THREAD

**Issue by Araxeus** [nushell/nushell#8467]
### Question

Here's my command:
```nu
def-env lsi [...path: string] { 
     let output = (ls-interactive ($path | str collect ' ')) 
     cd ( 
         if ($output | is-empty) { $env.PWD }  
         else { $output } 
     ) 
 } 
```

[ls-interactive](https://github.com/Araxeus/ls-interactive) is an interactive file chooser, but now suddenly it doesn't seem to show its stdout while it's running or something
I'm not sure what's going on, it just doesn't display anything

It was working fine on <=0.76.0

### Additional context and details

I have read the release notes and the [breaking changes](https://www.nushell.sh/blog/2023-03-14-nushell_0_77.html#breaking-changes) specifically, but I don't see anything that I use here? what am I missing? is this a bug?

**Comment by Araxeus** [comment]
I isolated the problem to the Parentheses `()` operator

just go to the command line and run `ls-interactive` - its all working fine
now run `(ls-interactive)` and there is nothing displayed at all

this wasn't the behavior in v0.76.0 and I think this might be a bug actually

**Comment by Araxeus** [comment]
Ok lol I have the easiest reproducible case:

run 
```nu
(cargo check)
```

this is definitely a bug

**Comment by sholderbach** [comment]
Could this be related to the changes in #8204 that prompt the issue #8472 ?

**Comment by Araxeus** [comment]
Fixed by #8475 - released in hotfix https://github.com/nushell/nushell/releases/tag/0.77.1
